# German po for libgda
# Copyright (C) 2000 Free Software Foundation, Inc.
# Christian Meyer <cm@ggtt.de>, 2000, 2001.
# Kai Lahmann <kl@linuxfaqs.de>, 2000.
#
msgid ""
msgstr ""
"Project-Id-Version: libgda-0.2.3-cvs\n"
"POT-Creation-Date: 2001-09-04 02:39+0200\n"
"PO-Revision-Date: 2001-04-29 03:52+0200\n"
"Last-Translator: Christian Meyer <chrisime@gnome.org>\n"
"Language-Team: German <gnome-de@gnome.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=ISO-8859-1\n"
"Content-Transfer-Encoding: 8bit\n"

#: lib/gda-client/gda-connection-pool.c:257
#, c-format
msgid "could not open connection to %s"
msgstr "Vebindung zu %s konnte nicht geöffnet werden"

#: lib/gda-client/gda-connection-pool.c:269
#, c-format
msgid "Data source %s not found"
msgstr "Datenquelle %s nicht gefunden"

#: lib/gda-client/gda-connection.c:428
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Datenbank '%s' nicht in der Systemkonfiguration gefunden"

#: lib/gda-client/gda-connection.c:431 lib/gda-client/gda-connection.c:447
msgid "[GDA Client Library]"
msgstr ""

#: lib/gda-client/gda-connection.c:446
msgid "Could not open CORBA factory"
msgstr "CORBA-Einheit konnte nicht geöffnet werden"

#: lib/gda-common/gda-corba.c:73
#, c-format
msgid "CORBA System exception: %s"
msgstr "CORBA-System-Ausnahmefehler: %s"

#: lib/gda-common/gda-corba.c:108
msgid "True"
msgstr "Wahr"

#: lib/gda-common/gda-corba.c:109
msgid "False"
msgstr "Falsch"

#: lib/gda-common/gda-error.c:121
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Ein Fehler ist in dem CORBA-System aufgetreten."

#: lib/gda-common/gda-error.c:132
#, fuzzy
msgid "[CORBA User Exception]"
msgstr "CORBA Anwenderfehler: %s"

#: lib/gda-common/gda-error.c:167
msgid "Unknown CORBA exception for connection"
msgstr ""

#: lib/gda-common/gda-field.c:640
msgid "<NULL>"
msgstr "<NULL>"

#: lib/gda-common/gda-field.c:644
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Unbekannter GDA-Typ(NULL)>"

#: lib/gda-common/gda-field.c:651
msgid "TRUE"
msgstr "WAHR"

#: lib/gda-common/gda-field.c:653
msgid "FALSE"
msgstr "FALSCH"

#: lib/gda-common/gda-init.c:35
msgid "Attempt to initialize an already initialized client"
msgstr "Versuch, einen bereits initialisierten Client zu initialisieren"

#: lib/gda-common/gda-init.c:44 lib/gda-server/gda-server-init.c:118
#, fuzzy
msgid "Could not initialize Bonobo"
msgstr "ct_diag() konnte nicht initialisiert werden"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Protokoll wird gespeichert...x"

#. compose message
#: lib/gda-common/gda-log.c:67
msgid "ERROR: "
msgstr "FEHLER:"

#: lib/gda-common/gda-log.c:67
msgid "MESSAGE: "
msgstr "MELDUNG:"

#: lib/gda-common/gda-util.c:113
#, fuzzy, c-format
msgid "Could not open file %s"
msgstr "Lokale Struktur konnte nicht auf '%s' gesetzt werden"

#: lib/gda-common/gda-util.c:138
#, fuzzy, c-format
msgid "Could not create file %s"
msgstr "Das Feld konnte nicht erreicht werden"

#: lib/gda-common/gda-xml-database.c:168
msgid "bad formed document"
msgstr ""

#: lib/gda-common/gda-xml-database.c:270
#, c-format
msgid "table %s already exists"
msgstr "Tabelle %s existiert schon"

#: lib/gda-common/gda-xml-database.c:452
#, c-format
msgid "%p is not a valid table"
msgstr "%p ist keine gültige Tabelle"

#: lib/gda-common/gda-xml-item.c:371
#, fuzzy, c-format
msgid "Item with id %s not found"
msgstr "Datenquelle %s nicht gefunden"

#: lib/gda-server/gda-server-connection.c:392
#, fuzzy
msgid "Connection being closed"
msgstr "Die Verbindung scheint tot zu sein"

#: lib/gda-server/gda-server-connection.c:394
msgid "This connection is being closed, so all listeners are released"
msgstr ""

#: lib/gda-server/gda-server-connection.c:817
msgid "Could not get pointer to server implementation"
msgstr "Zeiger konnte nicht auf die Server-Implementiertung gesetzt werden"

#: lib/gda-server/gda-server-init.c:43 lib/gda-server/gda-server-init.c:49
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Signal %d empfangen, Abbruch"

#: lib/gda-server/gda-server-init.c:53
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Signal %d empfangen. Wird ignoriert"

#: lib/gda-server/gda-server-init.c:102
msgid "Attempt to initialize an already initialized provider"
msgstr "Versuch, einen bereits initialisierten Anbieter zu initialisieren"

#: lib/gda-server/gda-server-recordset.c:53
#: lib/gda-server/gda-server-recordset.c:61
#: lib/gda-server/gda-server-recordset.c:101
#: lib/gda-server/gda-server-recordset.c:110
#, c-format
msgid "%s not implemented"
msgstr "%s nicht implementiert"

#: lib/gda-server/gda-server.c:130
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Anbieter %s ohne Implementierungsfunktionen wird gestartet"

#: lib/gda-server/gda-server.c:214
#, c-format
msgid "CORBA system exception %s"
msgstr "CORBA-System-Ausnahmefehler %s"

#: lib/gda-server/gda-server.c:218
#, c-format
msgid "CORBA user exception: %s"
msgstr "CORBA Anwenderfehler: %s"

#: providers/gda-default-server/GNOME_Database_Provider_Default.server.in.h:1
#, fuzzy
msgid "Default provider connection factory"
msgstr "Vebindung zu %s konnte nicht geöffnet werden"

#: providers/gda-default-server/GNOME_Database_Provider_Default.server.in.h:2
#, fuzzy
msgid "GDA Default Datasource Access"
msgstr "GDA-Datasource-Access für ODBC"

#: providers/gda-default-server/gda-default-connection.c:127
msgid "You must specify a directory for the database to be stored"
msgstr ""

#: providers/gda-default-server/gda-default-connection.c:235
#: providers/gda-mysql-server/gda-mysql-connection.c:282
#: providers/gda-oracle-server/gda-oracle-connection.c:334
#: providers/gda-postgres-server/gda-postgres-connection.c:487
#, fuzzy
msgid "Unknown schema type"
msgstr "Unbekannter ORACLE-Typ %ld"

#: providers/gda-default-server/main-default.c:69
#, fuzzy
msgid "Could not register Default provider implementation"
msgstr "Die Mail-Anbieter-Implementierung konnte nicht registriert werden"

#: providers/gda-ldap-server/GNOME_Database_Provider_LDAP.server.in.h:1
msgid "GDA Datasource Access for LDAP"
msgstr "GDA-Datasource-Access für LDAP"

#: providers/gda-ldap-server/GNOME_Database_Provider_LDAP.server.in.h:2
msgid "LDAP provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:288
#: providers/gda-ldap-server/gda-ldap-connection.c:293
#: providers/gda-mdb-server/gda-mdb-connection.c:372
#: providers/gda-mysql-server/gda-mysql-connection.c:486
#: providers/gda-odbc-server/gda-odbc-connection.c:624
#: providers/gda-odbc-server/gda-odbc-connection.c:657
#: providers/gda-oracle-server/gda-oracle-connection.c:479
#: providers/gda-postgres-server/gda-postgres-connection.c:768
#: providers/gda-sybase-server/gda-sybase-error.c:127
#: providers/gda-tds-server/gda-tds-error.c:123
#, c-format
msgid "error '%s' at %s"
msgstr "Fehler '%s' bei %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:294
#: providers/gda-ldap-server/gda-ldap-connection.c:298
#: providers/gda-ldap-server/gda-ldap-connection.c:299
#: providers/gda-mdb-server/gda-mdb-connection.c:377
#: providers/gda-mysql-server/gda-mysql-connection.c:491
#: providers/gda-mysql-server/gda-mysql-connection.c:492
#: providers/gda-odbc-server/gda-odbc-connection.c:626
#: providers/gda-odbc-server/gda-odbc-connection.c:628
#: providers/gda-odbc-server/gda-odbc-connection.c:659
#: providers/gda-odbc-server/gda-odbc-connection.c:661
#: providers/gda-oracle-server/gda-oracle-connection.c:484
#: providers/gda-oracle-server/gda-oracle-connection.c:485
#: providers/gda-postgres-server/gda-postgres-connection.c:776
#: providers/gda-postgres-server/gda-postgres-connection.c:777
#: providers/gda-primebase-server/gda-primebase-connection.c:375
#: providers/gda-primebase-server/gda-primebase-connection.c:376
#: providers/gda-sybase-server/gda-sybase-error.c:136
#: providers/gda-sybase-server/gda-sybase-error.c:137
#: providers/gda-tds-server/gda-tds-error.c:132
#: providers/gda-tds-server/gda-tds-error.c:133
msgid "Not available"
msgstr "Nicht verfügbar"

#: providers/gda-interbase-server/gda-interbase-connection.c:295
#: providers/gda-ldap-server/gda-ldap-connection.c:300
#: providers/gda-mdb-server/gda-mdb-connection.c:378
#: providers/gda-mysql-server/gda-mysql-connection.c:493
#: providers/gda-oracle-server/gda-oracle-connection.c:486
#: providers/gda-postgres-server/gda-postgres-connection.c:778
#: providers/gda-primebase-server/gda-primebase-connection.c:377
#: providers/gda-sybase-server/gda-sybase-error.c:138
#: providers/gda-tds-server/gda-tds-error.c:134
msgid "error"
msgstr "Fehler"

#: providers/gda-ldap-server/main-ldap.c:73
msgid "Could not register LDAP provider implementation"
msgstr "LDAP-Anbieter Implementierung konnte nicht registriert werden"

#: providers/gda-interbase-server/GNOME_Database_Provider_InterBase.server.in.h:1
msgid "GDA Datasource Access for Interbase"
msgstr "GDA-Datasource-Access für Interbase"

#: providers/gda-interbase-server/GNOME_Database_Provider_InterBase.server.in.h:2
msgid "InterBase provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:286
#, c-format
msgid "Error code %ld"
msgstr "Fehlercode %ld"

#: providers/gda-interbase-server/main-interbase.c:74
msgid "Could not register INTERBASE provider implementation"
msgstr "INTERBASE-Anbieter Implementierung konnte nicht registriert werden"

#: providers/gda-mail-server/GNOME_Database_Provider_Mail.server.in.h:1
msgid "GDA Datasource Access for Mail"
msgstr "GDA-Datasource-Access für Mail"

#: providers/gda-mail-server/GNOME_Database_Provider_Mail.server.in.h:2
msgid "Mail provider connection factory"
msgstr ""

#: providers/gda-mail-server/gda-mail-connection.c:107
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-sybase-server/gda-sybase-connection.c:285
#: providers/gda-tds-server/gda-tds-connection.c:278
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "Unbehandelter SCHEMA_QTYPE %d"

#: providers/gda-mail-server/main-mail.c:74
msgid "Could not register Mail provider implementation"
msgstr "Die Mail-Anbieter-Implementierung konnte nicht registriert werden"

#: providers/gda-mdb-server/GNOME_Database_Provider_MDB.server.in.h:1
msgid "GDA Datasource Access for MDB (MS Access) files"
msgstr "GDA-Datasource-Access für MDB (MS Access) Dateien"

#: providers/gda-mdb-server/GNOME_Database_Provider_MDB.server.in.h:2
msgid "MDB provider connection factory"
msgstr ""

#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:126
#: providers/gda-sybase-server/gda-sybase-command.c:141
#: providers/gda-tds-server/gda-tds-command.c:120
#, c-format
msgid "executing command '%s'"
msgstr "Befehl '%s' wird ansgeführt"

#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:131
#, c-format
msgid "info '%s'"
msgstr "Info '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: Sie konnen keinen UNIX_SOCKET anbieten, wenn Sie auch einen PORT und "
"einen HOST angeben. Bitte entfernen Sie entweder die PORT- und die HOST-"
"Optionen, oder entfernen Sie die UNIX_SOCKET-Optionen"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr "%s%d: Sie müssen einen Dateinamen für die MS-Access-Datenbank"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Verbindung mit Dateiname=%s, Benutzer=%s, Passwort=%s wird geöffnet"

#: providers/gda-mdb-server/gda-mdb-connection.c:427
msgid "mysql_recset is NULL"
msgstr "mysql_recset ist NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:469
#: providers/gda-mysql-server/gda-mysql-connection.c:838
#: providers/gda-sybase-server/gda-sybase-connection.c:464
#: providers/gda-tds-server/gda-tds-connection.c:448
msgid "No table name given for SCHEMA_COLS"
msgstr "Kein Tabellenname für SCHEMA_COLS angegeben"

#: providers/gda-mdb-server/main-mdb.c:72
#: providers/gda-mysql-server/main-mysql.c:74
msgid "Could not register MySQL provider implementation"
msgstr "Die MySQL-Anbieter-Implementierung kann nicht registriert werden"

#: providers/gda-mysql-server/GNOME_Database_Provider_MySQL.server.in.h:1
msgid "GDA Datasource Access for MySQL"
msgstr "GDA-Datasource-Access für MySQL"

#: providers/gda-mysql-server/GNOME_Database_Provider_MySQL.server.in.h:2
msgid "MySQL provider connection factory"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:174
#, fuzzy
msgid ""
"You cannot provide a UNIX_SOCKET if you also provide either a HOST or a "
"PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT options"
msgstr ""
"%s:%d: Sie konnen keinen UNIX_SOCKET anbieten, wenn Sie auch einen PORT und "
"einen HOST angeben. Bitte entfernen Sie entweder die PORT- und die HOST-"
"Optionen, oder entfernen Sie die UNIX_SOCKET-Optionen"

#: providers/gda-mysql-server/gda-mysql-connection.c:190
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, unix_socket=%"
"s"
msgstr ""
"Verbindung mit Benutzer=%s, Passwort=%s, Rechner=%s, Port=%s, Unix_Socket=%s "
"wird geöffnet"

#: providers/gda-mysql-server/gda-mysql-connection.c:616
msgid "Unknown"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:649
msgid "No MYSQL_Connection for GdaServerConnection"
msgstr ""

#: providers/gda-odbc-server/GNOME_Database_Provider_ODBC.server.in.h:1
msgid "GDA Datasource Access for ODBC"
msgstr "GDA-Datasource-Access für ODBC"

#: providers/gda-odbc-server/GNOME_Database_Provider_ODBC.server.in.h:2
msgid "ODBC provider connection factory"
msgstr ""

#: providers/gda-odbc-server/gda-odbc-connection.c:277
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: Unhebandelter SCHEMA_QTYPE %d\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:641
#: providers/gda-odbc-server/gda-odbc-connection.c:642
#: providers/gda-odbc-server/gda-odbc-connection.c:643
#: providers/gda-odbc-server/gda-odbc-connection.c:674
#: providers/gda-odbc-server/gda-odbc-connection.c:675
#: providers/gda-odbc-server/gda-odbc-connection.c:676
msgid "no text"
msgstr "Kein Text"

#: providers/gda-odbc-server/main-odbc.c:73
msgid "Could not register ODBC provider implementation"
msgstr "Die ODBC-Anbieter-Implementierung konnte nicht registriert werden"

#: providers/gda-oracle-server/GNOME_Database_Provider_Oracle.server.in.h:1
msgid "GDA Datasource Access for Oracle"
msgstr "GDA-Datasource-Access für Oracle"

#: providers/gda-oracle-server/GNOME_Database_Provider_Oracle.server.in.h:2
msgid "Oracle provider connection factory"
msgstr ""

#: providers/gda-oracle-server/gda-oracle-command.c:191
msgid "Could not allocate statement handle"
msgstr "`Statement handle' konnte nicht erreicht werden"

#: providers/gda-oracle-server/gda-oracle-command.c:197
msgid "command object being created without associated connection"
msgstr "Befehlsobjekt erstellt, ohne eine dazugehörige Verbindung zu haben"

#: providers/gda-oracle-server/gda-oracle-command.c:238
#, c-format
msgid "Executing command '%s'"
msgstr "Befehl '%s' wird ausgeführt"

#: providers/gda-oracle-server/gda-oracle-connection.c:117
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "Fehler in OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:123
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "Fehler in OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:129
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "Fehler in OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:418
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Unbekannter ORACLE-Typ %ld"

#: providers/gda-oracle-server/gda-oracle-recordset.c:118
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Unbekannter ORACLE-Datentyp in fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:156
msgid "recordset object being created without associated connection"
msgstr "Recordset-Objekt erstellt, ohne eine dazugehörige Verbindung zu haben"

#: providers/gda-oracle-server/main-oracle.c:69
msgid "Could not register ORACLE provider implementation"
msgstr "Die ORACLE-Anbieter-Implementiertung konnte nicht registriert werden"

#: providers/gda-postgres-server/GNOME_Database_Provider_Postgres.server.in.h:1
msgid "GDA Datasource Access for Postgres"
msgstr "GDA-Datasource-Access für Postgres"

#: providers/gda-postgres-server/GNOME_Database_Provider_Postgres.server.in.h:2
msgid "Postgres provider connection factory"
msgstr ""

#: providers/gda-postgres-server/gda-postgres-connection.c:889
#: providers/gda-postgres-server/gda-postgres-connection.c:1006
msgid "postgres_recset is NULL"
msgstr "postgres_recset ist NULL"

#: providers/gda-postgres-server/main-postgres.c:71
msgid "Could not register PostgreSQL provider implementation"
msgstr ""
"Die PostgreSQL-Anbieter-Implementiertung konnte nicht registriert werden"

#: providers/gda-sybase-server/GNOME_Database_Provider_Sybase.server.in.h:1
msgid "GDA Datasource Access for Sybase"
msgstr "GDA-Datasource-Access für Sybase"

#: providers/gda-sybase-server/GNOME_Database_Provider_Sybase.server.in.h:2
msgid "Sybase provider connection factory"
msgstr ""

#.      gda_server_error_make(error, 0, cnc, "connection seems to be dead");
#: providers/gda-sybase-server/gda-sybase-command.c:136
#: providers/gda-tds-server/gda-tds-command.c:116
msgid "Connection seems to be dead"
msgstr "Die Verbindung scheint tot zu sein"

#: providers/gda-sybase-server/gda-sybase-command.c:153
#: providers/gda-tds-server/gda-tds-command.c:130
msgid "Filling command structure failed\n"
msgstr "Füllen der Befehlsstruktur fehlgeschlagen\n"

#: providers/gda-sybase-server/gda-sybase-command.c:162
#: providers/gda-tds-server/gda-tds-command.c:137
msgid "sending cmd failed"
msgstr "Senden von cmd fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-command.c:169
#: providers/gda-tds-server/gda-tds-command.c:144
msgid "Allocating recordset failed"
msgstr "Erreichen des Aufnahmesatzes fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-command.c:181
#: providers/gda-tds-server/gda-tds-command.c:153
msgid "Getting recordset userdata failed"
msgstr "Erreichen der Aufnahmesatz-Benutzerdaten fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-command.c:241
#: providers/gda-tds-server/gda-tds-command.c:210
msgid "Command execution finished, all results processed"
msgstr "Befehlsausführung beendet, alle Ergebnisse abgearbeitet"

#: providers/gda-sybase-server/gda-sybase-command.c:245
#: providers/gda-tds-server/gda-tds-command.c:214
msgid "Command failed"
msgstr "Befehl fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-command.c:250
#: providers/gda-tds-server/gda-tds-command.c:219
msgid "Execution of resultless command succeeded"
msgstr "Ausführen des ergebnislosen Befehls fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-command.c:268
#: providers/gda-tds-server/gda-tds-command.c:237
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() gab %x zurück"

#: providers/gda-sybase-server/gda-sybase-command.c:280
#: providers/gda-tds-server/gda-tds-command.c:249
msgid "No further results, command completly succeeded."
msgstr "Keine weiteren Ergebnisse, Befehl erfolgreich beendet."

#: providers/gda-sybase-server/gda-sybase-command.c:285
#: providers/gda-tds-server/gda-tds-command.c:254
msgid "An error occured during command execution."
msgstr "Ein Fehler trat beim Ausführen des Befehls auf."

#: providers/gda-sybase-server/gda-sybase-command.c:291
#: providers/gda-tds-server/gda-tds-command.c:258
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"Ausführen des Befehls hat ein unbekanntes Ergebnis zurückgegeben, "
"möglicherweise ein Fehler"

#: providers/gda-sybase-server/gda-sybase-command.c:304
#: providers/gda-tds-server/gda-tds-command.c:269
msgid "No data returned, freeing recordset structure"
msgstr "Keine Daten zurückgegeben, die Struktur wird gelöscht"

#: providers/gda-sybase-server/gda-sybase-command.c:336
#: providers/gda-sybase-server/gda-sybase-command.c:348
#: providers/gda-tds-server/gda-tds-command.c:300
#: providers/gda-tds-server/gda-tds-command.c:310
msgid "Freeing command structure failed"
msgstr "Löschen der Befehlsstruktur fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-connection.c:171
#, fuzzy
msgid "Not enough memory for sybase_Connection"
msgstr "Nicht genug Speicher für primebase_Error Struktur"

#: providers/gda-sybase-server/gda-sybase-connection.c:453
#: providers/gda-tds-server/gda-tds-connection.c:437
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: table_name '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:470
#: providers/gda-tds-server/gda-tds-connection.c:454
msgid "Could not allocate command structure"
msgstr "Befehlsstruktur konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:493
#: providers/gda-tds-server/gda-tds-connection.c:477
msgid "Could not allocate enough memory for query"
msgstr "Es konnte nicht genug Speicher für die Anfrage reserviert werden"

#.          gda_server_error_make(error, NULL, cnc, __PRETTY_FUNCTION__);
#: providers/gda-sybase-server/gda-sybase-connection.c:508
#: providers/gda-tds-server/gda-tds-connection.c:492
msgid "Could not get recordset in schema_cols"
msgstr "Aufnahmesatz in schema_cols konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:557
#: providers/gda-tds-server/gda-tds-connection.c:541
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: Ungültiger Konstantentyp %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:567
#: providers/gda-tds-server/gda-tds-connection.c:551
msgid "Could not allocate cmd structure"
msgstr "Die Befehlsstruktur konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:633
#: providers/gda-tds-server/gda-tds-connection.c:617
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: Ungültiger Konstantentyp %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:695
#: providers/gda-tds-server/gda-tds-connection.c:679
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Ungültiger oder nicht unterstützter Konstantentyp: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:738
#: providers/gda-tds-server/gda-tds-connection.c:722
msgid "Could not allocate GdaServerCommand"
msgstr "GdaServerCommand konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:769
#: providers/gda-sybase-server/gda-sybase-connection.c:774
#: providers/gda-tds-server/gda-tds-connection.c:752
#: providers/gda-tds-server/gda-tds-connection.c:757
#, c-format
msgid "connection status: %x"
msgstr "Verbindungsstatus: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:770
#: providers/gda-tds-server/gda-tds-connection.c:753
#, c-format
msgid "return code: %x"
msgstr "Rückgabewert: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:808
#: providers/gda-tds-server/gda-tds-connection.c:788
msgid "attempting reconnection"
msgstr "Verbindung wird erneut versucht"

#: providers/gda-sybase-server/gda-sybase-connection.c:938
#, c-format
msgid "Reconnected to '%s'"
msgstr "Verbunden mit '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:940
#: providers/gda-tds-server/gda-tds-connection.c:890
#, c-format
msgid "Connected to %s"
msgstr "Verbunden mit '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:952
#: providers/gda-tds-server/gda-tds-connection.c:904
#, c-format
msgid "Selected database '%s'"
msgstr "Gewählte Datenbank '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:981
#: providers/gda-tds-server/gda-tds-connection.c:930
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Die Befehlsstruktur konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:990
#: providers/gda-tds-server/gda-tds-connection.c:939
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Speicher für den Befehlsstring konnte nicht alloziiert werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:999
#: providers/gda-tds-server/gda-tds-connection.c:947
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-connection.c:1010
#: providers/gda-tds-server/gda-tds-connection.c:957
#, c-format
msgid "%s: sending command failed"
msgstr "%s: Senden des Befehls fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-connection.c:1021
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: Auswählen der Datenbank fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-connection.c:1038
#, c-format
msgid "%s: use command failed"
msgstr "%s: use-Befehl fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-connection.c:1066
#: providers/gda-tds-server/gda-tds-connection.c:1020
msgid "Could not allocate locale structure"
msgstr "Die lokale Struktur kann nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:1072
#: providers/gda-tds-server/gda-tds-connection.c:1026
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Lokale Struktur konnte nicht auf '%s' gesetzt werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:1079
#: providers/gda-tds-server/gda-tds-connection.c:1033
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr ""
"Die lokale Struktur konnte nicht so eingestellt werden, dass sie '%s' ist"

#: providers/gda-sybase-server/gda-sybase-connection.c:1085
#: providers/gda-tds-server/gda-tds-connection.c:1039
msgid "Could not drop locale structure"
msgstr "Die lokale Struktur konnte nicht gelöscht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:1099
#: providers/gda-tds-server/gda-tds-connection.c:1052
#, c-format
msgid "Fatal error: %s\n"
msgstr "Schwerer Fehler: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:73
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:85
#: providers/gda-tds-server/gda-tds-error.c:81
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""
"Kein Fehlercode angegeben, obwohl %s aufgerufen wurde. Dies könnte ein "
"Fehler sein."

#: providers/gda-sybase-server/gda-sybase-error.c:161
msgid "cs_diag() initialized"
msgstr "cs_diag() initialisiert"

#: providers/gda-sybase-server/gda-sybase-error.c:165
msgid "Could not initialize cs_diag()"
msgstr "cs_diag() konnte nicht initialisiert werden"

#: providers/gda-sybase-server/gda-sybase-error.c:175
msgid "ct_diag() initialized"
msgstr "ct_diag() initialisiert"

#: providers/gda-sybase-server/gda-sybase-error.c:179
msgid "Could not initialize ct_diag()"
msgstr "ct_diag() konnte nicht initialisiert werden"

#: providers/gda-sybase-server/gda-sybase-error.c:215
#, c-format
msgid "%s could not find server implementation"
msgstr "%s konnte die Server-Implementierung nicht finden"

#: providers/gda-sybase-server/gda-sybase-error.c:222
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s konnte die cnc-Benutzerdaten nicht erhalten"

#: providers/gda-sybase-server/gda-sybase-error.c:234
msgid "Could not get # of messages"
msgstr "Die Anzahl der Meldungen konnte nicht erhalten werden"

#: providers/gda-sybase-server/gda-sybase-error.c:241
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d ctlib-Meldungen zu empfangen"

#: providers/gda-sybase-server/gda-sybase-error.c:246
#, c-format
msgid "%d ctlib message to fetch"
msgstr "ctlib-Meldung %d zu empfangen"

#: providers/gda-sybase-server/gda-sybase-error.c:284
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "Fehler ist Nullzeiger in gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:310
#: providers/gda-sybase-server/gda-sybase-error.c:382
#: providers/gda-sybase-server/gda-sybase-error.c:429
msgid "Could not get # of server messages"
msgstr "Die Anzahl der Server-Meldungen konnte nicht erhalten werden"

#: providers/gda-sybase-server/gda-sybase-error.c:318
#, c-format
msgid "Fetching %d client messages"
msgstr "%d Client-Meldungen werden geholt"

#: providers/gda-sybase-server/gda-sybase-error.c:321
#, c-format
msgid "Fetching %d client message"
msgstr "Client-Meldung %d wird geholt"

#: providers/gda-sybase-server/gda-sybase-error.c:330
#, c-format
msgid "Could not get client msg # %d"
msgstr "Die Client-Meldungen %d konnte nicht empfangen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:349
msgid "Could not get # of client messages"
msgstr "Die Anzahl der Client-Meldungen konnte nicht ermittelt werden"

#: providers/gda-sybase-server/gda-sybase-error.c:357
msgid "Could not clear client messages"
msgstr "Client-Meldungen konnte nicht gelöscht werden"

#: providers/gda-sybase-server/gda-sybase-error.c:390
#, c-format
msgid "Fetching %d server messages"
msgstr "%d Server-Meldungen werden geholt"

#: providers/gda-sybase-server/gda-sybase-error.c:393
#, c-format
msgid "Fetching %d server message"
msgstr "Server-Meldung %d werden geholt"

#: providers/gda-sybase-server/gda-sybase-error.c:402
#, c-format
msgid "Could not get server msg # %d"
msgstr "Die Server-Meldung %d konnte nicht empfangen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:437
msgid "Could not clear server messages"
msgstr "Die Server-Meldungen konnte nicht gelöscht werden"

#: providers/gda-sybase-server/gda-sybase-error.c:459
#: providers/gda-sybase-server/gda-sybase-error.c:499
msgid "Could not get # of cslib messages"
msgstr "Die Anzahl der cslib-Meldungen konnte nicht empfangen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:467
#, c-format
msgid "Fetching %d cslib messages"
msgstr "%d cslib-Meldungen werden geholt"

#: providers/gda-sybase-server/gda-sybase-error.c:470
#, c-format
msgid "Fetching %d cslib message"
msgstr "cslib-Meldung %d wird geholt"

#: providers/gda-sybase-server/gda-sybase-error.c:480
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Die cslib-Meldung %d konnte nicht empfangen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:507
msgid "Could not clear cslib messages"
msgstr "Die cslib-Meldungen konnte nicht gelöscht werden"

#: providers/gda-sybase-server/gda-sybase-error.c:641
#: providers/gda-tds-server/gda-tds-error.c:269
msgid "Server message:"
msgstr "Server-Nachricht:"

#: providers/gda-sybase-server/gda-sybase-error.c:649
#: providers/gda-tds-server/gda-tds-error.c:277
msgid "Client message:"
msgstr "Client-Nachricht:"

#: providers/gda-sybase-server/gda-sybase-error.c:656
#: providers/gda-tds-server/gda-tds-error.c:284
msgid "CS-Library error:"
msgstr "CS-Bibliotheksfehler:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:198
#: providers/gda-tds-server/gda-tds-recordset.c:194
#, c-format
msgid "fetch error at row %d"
msgstr "Fehler in Zeile %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:205
#: providers/gda-tds-server/gda-tds-recordset.c:201
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA in Zeile %d; Abbruch"

#: providers/gda-sybase-server/gda-sybase-recordset.c:213
#: providers/gda-tds-server/gda-tds-recordset.c:208
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Fehler (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:386
#: providers/gda-tds-server/gda-tds-recordset.c:382
msgid "Failed fetching # of columns"
msgstr "Die Anzahl der Spalten konnte nicht geholt werden"

#: providers/gda-sybase-server/gda-sybase-recordset.c:393
#: providers/gda-tds-server/gda-tds-recordset.c:390
#, c-format
msgid "Counted %d columns"
msgstr "%d Spalten gezählt"

#: providers/gda-sybase-server/gda-sybase-recordset.c:400
#: providers/gda-tds-server/gda-tds-recordset.c:397
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s konnte `datafmt info' nicht erreichen"

#: providers/gda-sybase-server/gda-sybase-recordset.c:410
#: providers/gda-tds-server/gda-tds-recordset.c:407
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s konnte die Datenfelder nicht erreichen"

#: providers/gda-sybase-server/gda-sybase-recordset.c:428
#: providers/gda-tds-server/gda-tds-recordset.c:425
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe in Zeile %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:435
#: providers/gda-tds-server/gda-tds-recordset.c:432
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe in Zeile %d fehlgeschlagen"

#: providers/gda-sybase-server/gda-sybase-recordset.c:442
#: providers/gda-tds-server/gda-tds-recordset.c:439
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:454
#: providers/gda-tds-server/gda-tds-recordset.c:450
msgid "could not allocate data holder"
msgstr "Der Datenhalter konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-recordset.c:467
#: providers/gda-tds-server/gda-tds-recordset.c:463
msgid "could not ct_bind data holder to column"
msgstr "Datenhalter konnte nicht an Spalte gebunden (ct_bind) werden"

#: providers/gda-sybase-server/gda-sybase-recordset.c:474
#: providers/gda-tds-server/gda-tds-recordset.c:470
msgid "illegal type detected, aborting"
msgstr "Unzulässiger Typ entdeckt, Abbruch"

#: providers/gda-sybase-server/gda-sybase-recordset.c:481
#: providers/gda-tds-server/gda-tds-recordset.c:477
msgid "could not allocate field"
msgstr "Das Feld konnte nicht erreicht werden"

#: providers/gda-sybase-server/main-sybase.c:78
msgid "Could not register gda-sybase provider implementation"
msgstr ""
"Die gda-sybase-Anbieter-Implementiertung konnte nicht registriert werden"

#: providers/gda-primebase-server/GNOME_Database_Provider_Primebase.server.in.h:1
msgid "GDA Datasource Access for primebase"
msgstr "GDA-Datasource-Access für 'primebase'"

#: providers/gda-primebase-server/gda-primebase-connection.c:157
#, c-format
msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
msgstr "Verbindung mit '%s@%s:%s' mit Verb.Param. nicht möglich: '%s'"

#: providers/gda-primebase-server/gda-primebase-connection.c:163
#, c-format
msgid "%2ld: Connected to '%s@%s:%s' with connparm: '%s'"
msgstr "%2ld: Verbunden mit '%s@%s:%s' mit Verb.Param.: '%s'"

#: providers/gda-primebase-server/gda-primebase-connection.c:171
msgid "Could not get PBIConnectionInfo(), disconnecting"
msgstr ""
"PBIConnectionInfo() konnte nicht empfangen werden, Verbindung wird getrennt"

#: providers/gda-primebase-server/gda-primebase-connection.c:176
#, c-format
msgid "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"
msgstr "%2ld: Verb.Info: %s@%s, Cl v%x, Srv '%s' v%x, Typ '%s'"

#: providers/gda-primebase-server/gda-primebase-connection.c:390
msgid "Not enough memory for primebase_Error structure."
msgstr "Nicht genug Speicher für primebase_Error Struktur"

#: providers/gda-primebase-server/gda-primebase-connection.c:397
#, c-format
msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
msgstr "%2ld: FEHLER: %ld (%ld) : %s.\n"

#: providers/gda-primebase-server/main-primebase.c:73
msgid "Could not register PRIMEBASE provider implementation"
msgstr "PRIMEBASE-Anbieter Implementierung konnte nicht registriert werden"

#: providers/gda-tds-server/GNOME_Database_Provider_TDS.server.in.h:1
msgid "GDA Datasource Access for Sybase/MS SQL"
msgstr "GDA-Datasource-Acess für Sybase/MS SQL"

#: providers/gda-tds-server/GNOME_Database_Provider_TDS.server.in.h:2
msgid "TDS provider connection factory"
msgstr ""

#: providers/gda-tds-server/gda-tds-connection.c:160
#, fuzzy
msgid "Not enough memory for tds_Connection"
msgstr "Nicht genug Speicher für primebase_Error Struktur"

#. Fixme: replace "dummy" with buf when CS_GET CS_SERVERNAME works
#: providers/gda-tds-server/gda-tds-connection.c:888
#, c-format
msgid "Reconnected to %s"
msgstr "Wieder verbunden mit %s"

#: providers/gda-tds-server/gda-tds-error.c:70
#, c-format
msgid "gda_tds_error_make(%p,%p,%p,%p)"
msgstr "gda_tds_error_make(%p,%p,%p,%p)"

#: providers/gda-tds-server/main-tds.c:78
msgid "Could not register gda-tds provider implementation"
msgstr "Die gda-tds-Anbieter-Implementiertung konnte nicht registriert werden"

#: tools/gda-run.c:102
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: Fehlkonfigurierter DSN-Eintrag"

#: tools/gda-run.c:116
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: Datenquelle '%s' nicht gefunden\n"

#: tools/templates/main-srv.c.tmpl:60
#, fuzzy
msgid "Could not register <PROVIDER-NAME> provider implementation"
msgstr "PRIMEBASE-Anbieter Implementierung konnte nicht registriert werden"

#~ msgid "%s: The server didn't respond."
#~ msgstr "%s: Der Server antwortet nicht."

#~ msgid "thread is already running"
#~ msgstr "Thread läuft schon"

#~ msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
#~ msgstr "impl_GDA_Connection_open: Verbindung mit DSN öffnen: %s"

#~ msgid "%s: an error ocurred while fetching data"
#~ msgstr "%s: Ein Fehler trat auf, während die Daten empfangen werden"

#~ msgid ""
#~ "OAF doesn't know about our IID; indicates broken installation. Exiting..."
#~ msgstr ""
#~ "OAF weiß nichts über ihre IID. Das deutet auf defekte Installation hin. "
#~ "Abbruch..."

#~ msgid ""
#~ "Another instance of this provider is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Eine andere Instanz dieses Anbieters ist bereits mit OAF registriert. "
#~ "Abbruch..."

#~ msgid "Unknown error registering this provider with OAF. Exiting"
#~ msgstr ""
#~ "Unbekannter Fehler beim Registrieren dieses Anbieters mit OAF. Abbruch"

#~ msgid "Registered with ID = %s"
#~ msgstr "Registriert mit ID = %s"

#~ msgid "CORBA exception unloading report engine: %s"
#~ msgstr "CORBA-Ausnahme, Berichtsmodul wird entfernt: %s"

#~ msgid "GDA Distributed Report Engine"
#~ msgstr "GDA-Distributed-Report-Engine"

#~ msgid "Received signal %d, shutting down."
#~ msgstr "Signal %d wurde erhalten, Abbruch."

#~ msgid "Received signal %d, shutting down cleanly"
#~ msgstr "Signal %d wurde erhalten, fehlerfreies Herunterfahren"

#~ msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
#~ msgstr ""
#~ "OAF konnte nicht initialisiert werden: Bitte berichten Sie diesen Fehler "
#~ "an die OAF-Entwickler"

#~ msgid "Could not activate report engine"
#~ msgstr "Die Berichts-Engine konnte nicht aktiviert werden"

#~ msgid ""
#~ "Another instance of the report engine is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Eine andere Instalz dieses Anbieters ist bereits mit OAF registriert. "
#~ "Beenden..."

#~ msgid "Unknown error registering report engine with OAF. Exiting"
#~ msgstr ""
#~ "Unbekannter Fehler beim Registrieren dieses Anbieters mit OAF. Beenden"

#~ msgid "Registered report engine with ID = %s"
#~ msgstr "Berichtsmodul registriert mit ID = %s"

#~ msgid "schema_tables: table name = '%s'"
#~ msgstr "schema_tables: Tabellenname = '%s'"

#~ msgid "schema_tables: table_schema = '%s'"
#~ msgstr "schema_tables: table_schema = '%s'"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
#~ msgstr "schema_procedures: proc_catalog = '%s' UNGENUTZT!\n"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
#~ msgstr "schema_procedures: proc_catalog = '%s' UNGENUTZT!"

#~ msgid "database name"
#~ msgstr "Datenbankname"

#~ msgid "DATABASE"
#~ msgstr "DATENBANK"

#~ msgid "user name"
#~ msgstr "Benutzername"

#~ msgid "USER-NAME"
#~ msgstr "BENUTZERNAME"

#~ msgid "password"
#~ msgstr "Passwort"

#~ msgid "PASSWORD"
#~ msgstr "PASSWORT"

#~ msgid "file name"
#~ msgstr "Dateiname"

#~ msgid "FILE-NAME"
#~ msgstr "DATEINAME"
